docs/advanced-usage.md

```diff
@@ -19,12 +19,13 @@ Ensure that `package-lock.json` is always committed, use `npm ci` instead of `np
 
 ### Yarn
 
-Ensure that `yarn.lock` is always committed, pass `--frozen-lockfile` to `yarn install` when installing packages.
+Ensure that `yarn.lock` is always committed, pass `--frozen-lockfile` or `--immutable` to `yarn install` when installing packages.
```

Comment:
What's the difference between these options?  Is there reason to prefer one or the other?

Reply:
Hope more people test

Reply:
Thanks @xiaotiandada.  I say we just recommend `--immutable` then:

```suggestion
To ensure that `yarn.lock` is always committed, use `yarn install --immutable` when installing packages.
```